crates/sc-piece-cache/src/lib.rs

```diff
@@ -58,9 +64,31 @@ where
 {
     const KEY_PREFIX: &[u8] = b"piece_cache";
 
-    /// Create new instance
-    pub fn new(aux_store: Arc<AS>) -> Self {
-        Self { aux_store }
+    /// Create a new instance. Optional cache_size parameter could redefine the default cache size.
+    pub fn new(aux_store: Arc<AS>, cache_size: Option<ByteSize>) -> Self {
```

Comment:
Kept in bytes to preserve conversion logic in one place. Downgraded the log level to debug, IMHO it's useful to prevent the same errors in the future.